### PR TITLE
Update README and about-us

### PR DESCRIPTION
**Pull request by Harry-Gao-H**
README file is updated, but I am not sure it is 100% "match our project".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
+* This is **a brown-field team project done by NUS students**.<br>
   Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
+  * as an application management tool for HRs
   * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
+* The project simulates an ongoing software project for a desktop application (called _MrTechRecruiter_) used for managing job application.
```
Comment (BenjaminTan99) [inline README.md:9]: End of sentence should be 'job applications'.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC).
```
Comment (LMAOboxhack) [inline README.md:10]:
```suggestion
  * It is **written in OOP fashion**.
```
We are not providing a code base for anything, so can consider removing.

File: docs/AboutUs.md [hunk 3]
```diff
@@ -38,15 +38,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-<img src="images/johndoe.png" width="200px">
+<img src="images/harry-gao-h.png" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/harry-gao-h)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Code quality + Scheduling & Tracking
```
Comment (LMAOboxhack) [inline docs/AboutUs.md:48]:
```suggestion
* Responsibilities: Code quality, Scheduling & Tracking
```
Shall we standardize on commas?

**Review by LMAOboxhack — COMMENTED** [review]
A few small nits, otherwise LGTM!

**Pull request merged**